src/backend/bigquery/bigquery.test.ts

```diff
@@ -1,60 +1,231 @@
+        link: 'link',
+        selected_words_without_quotes: 'selectedWordsWithoutQuotes',
+      },
+    })
   })
+})
 
-  test("doesn't include linkClause when searchParams.linkSearchUrl is an empty string", () => {
-    const searchParams: SearchParams = makeParams()
-    const keywords: string[] = []
-    const excludedKeywords: string[] = []
+describe('[Function] sendInitQuery', () => {
+  it('should call bigQuery with the correct parameters', async () => {
+    ;(BigQuery.prototype.query as jest.Mock)
+      .mockResolvedValueOnce([
+        [{ locale: 'fr' }, { locale: 'en' }, { locale: 'de' }],
+      ])
+      .mockResolvedValueOnce([
+        [{ name: 'taxon1' }, { name: 'taxon2' }, { name: 'taxon3' }],
+      ])
+      .mockResolvedValueOnce([
+        [{ title: 'org1' }, { title: 'org2' }, { title: 'org3' }],
+      ])
+    const result = await sendInitQuery()
+
+    expect(BigQuery.prototype.query).toHaveBeenCalledTimes(3)
+
+    expect(BigQuery.prototype.query).toHaveBeenNthCalledWith(1, {
+      query: `
+        SELECT DISTINCT locale
+        FROM \`content.locale\`
+        `,
+      location: 'europe-west2',
+      params: {},
+    })
+    expect(BigQuery.prototype.query).toHaveBeenNthCalledWith(2, {
+      query: `
+        SELECT name
+        FROM \`search.taxon\`
+        `,
+      location: 'europe-west2',
+      params: {},
+    })
+    expect(BigQuery.prototype.query).toHaveBeenNthCalledWith(3, {
+      query: `
+        SELECT DISTINCT title
+        FROM \`graph.organisation\`
+        `,
+      location: 'europe-west2',
+      params: {},
+    })
 
-    const query = buildSqlQuery(searchParams, keywords, excludedKeywords)
+    expect(result).toEqual({
+      locales: ['', 'en', 'cy', 'fr', 'de'],
+      taxons: ['taxon1', 'taxon2', 'taxon3'],
+      organisations: ['org1', 'org2', 'org3'],
+    })
+  })
+})
 
-    expect(query).not.toContain(
-      `AND EXISTS
-        (
-          SELECT 1 FROM UNNEST (hyperlinks) AS link
-          WHERE CONTAINS_SUBSTR(link, @link)
-        )`
+describe('[Function] sendSearchQuery', () => {
+  const makeSearchParams = (options = {}) => ({
+    searchType: SearchType.Keyword,
+    selectedWords: 'keyword1 keyword2',
+    excludedWords: 'excluded1 excluded2',
+    selectedLocale: 'en',
+    selectedTaxon: 'taxon',
+    selectedOrganisation: 'organisation',
+    linkSearchUrl: 'link',
+    whereToSearch: { title: true, text: true },
+    combinator: Combinator.Any,
+    areaToSearch: SearchArea.Any,
+    caseSensitive: false,
+    ...options,
+  })
+  it('Calls the appropriate queries with default search type', async () => {
+    jest.spyOn(buildSqlQuery, 'buildSqlQuery').mockReturnValue('query')
+    ;(BigQuery.prototype.query as jest.Mock)
+      .mockResolvedValueOnce([
+        [{ locale: 'fr' }, { locale: 'en' }, { locale: 'de' }],
+      ])
+      .mockResolvedValueOnce([
+        [{ name: 'taxon1' }, { name: 'taxon2' }, { name: 'taxon3' }],
+      ])
```

Comment:
Are these required here?  They aren't representative of actual results in the `sendSearchQuery` function.

Reply:
That's the `sendInitQuery()` function.  This test is of the `sendSearchQuery()` function: https://github.com/alphagov/govuk-knowledge-graph-search/blob/54255efd943d08ab0454488893e1b3ddb5319155/src/backend/bigquery/bigquery.ts#L233-L237